models/Contact.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require('mongoose');
+const Language = require('./Language').schema;
+const ImageSet = require('../modules/common/ImageSet').schema;
+
+const ContactSchema = new mongoose.Schema({
+  phoneNumber: Number,
+  openHours: [Language],
+  address: {
```

Comment:
I think it is much better to convert `address` to an array because it is a huge possibility that Horondi will have more than one location due to the business scaling

Reply:
Done